lib/redmine/field_format.rb

```diff
@@ -114,7 +114,7 @@ def cast_single_value(custom_field, value, customized=nil)
       def target_class
         nil
       end
- 
+
```

Comment:
Could you keep this, just to reduce the change of merge conflicts when pulling from upstream?